auth-tokens/src/main/java/com/palantir/tokens/auth/BearerTokens.java

```diff
@@ -78,7 +78,7 @@ public static BearerToken fromPath(Path path) throws IOException {
         }
 
         LOGGER.warn("No token file found in any of the configured paths: {}.", paths);
-        return Optional.absent();
+        return Optional.empty();
```

Comment:
for service versions, sure. For libraries, especially one like this that's involved in diamond dependencies, we've been using the package numbering scheme.